contracts/Collector.sol

```diff
@@ -29,18 +32,33 @@ contract Collector is ICollector{
     external
     validShares(_partners)
     onlyOwner()
-    {
-        uint balance = token.balanceOf(address(this));
-        require(balance == 0, "can't update with balance > 0");
-    
+    noBalanceToShare()
```

Comment:
you don't the brackets if the modifier doesn't take any arguments

Reply:
Done.